src/controller/CommissionerDiscoveryController.h

```diff
@@ -392,6 +429,11 @@ class CommissionerDiscoveryController : public chip::Protocols::UserDirectedComm
     inline void SetPasscodeService(PasscodeService * passcodeService) { mPasscodeService = passcodeService; }
     inline PasscodeService * GetPasscodeService() { return mPasscodeService; }
 
+    /**
+     * Assign an AppInstallationService
+     */
+    inline void SetAppInstallationService(AppInstallationService * appInstallationService) { mAppInstallationService = appInstallationService; }
```

Comment:
Seems like this might be better located on the ContentAppPlatform

Reply:
Added the method to the pass code service